content/theme/characters/archetypes/outlaws/spice-dealer.md

```diff
@@ -2,5 +2,42 @@
 title: "Spice Dealer"
 date: 2020-11-14T18:45:44-08:00
 draft: false
+layout: archetype
+
+attributes:
+  awareness: 11
+  coordination: 11
+  knowledge: 9
+  physique: 7
+  presence: 7
```

Comment:
nitpick, it seems to me that a spice dealer probably has a higher presence score, owing to the need to establish and maintain interpersonal relationships.